moonworm/crawler/function_call_crawler.py

```diff
@@ -146,6 +147,7 @@ def __init__(
         self.contract_addresses = contract_addresses
         self.contract = Web3().eth.contract(abi=self.contract_abi)
         self.on_decode_error = on_decode_error
+        self.whitelisted_methods = [Web3().toHex(function_abi_to_4byte_selector(a)) for a in self.contract_abi]
```

Comment:
I suggest turning this into a set:
`self.whitelisted_methods = {Web3().toHex(function_abi_to_4byte_selector(a)) for a in self.contract_abi}`

O(1) lookups good for large ABIs in `process_transaction`.

Reply:
Okay thanks, did it.